### ISSUE TITLE
Map loader has weird bug with docking ports

### ISSUE THREAD

**Issue by Niknakflak** [tgstation/-tg-station#14591]
So this happens whenever any docking port is mapped and then placed with the map loader.  
It will always cut off these sections of any shuttle.
Here's a shuttle I made: The docking port is at the bottom with the correct bounds. 
http://i.imgur.com/ymAwjgZ.png
Here's what happens when you try to move it: 
http://imgur.com/h55avoM
It cuts off two tiles on the left and three on the right regardless of the bounds of the docking port.
I tried reversing the location of the docking port to the top and got similar results: 
http://imgur.com/UKBUAsF

This was tested a ton using this shuttle but also with just loading a single docking port with the map loader which also chopped off those tiles even if the only thing that was uploaded was a single docking port.  This was also tested with and without the shuttles having an area.

Long story short, the map loader does something completely weird when loading docking ports and it simply makes building any kind of movable shuttle impossible.

**Comment by tkdrg** [comment]
@RemieRichards

**Comment by RemieRichards** [comment]
Probably an off-by-1 somewhere.
Or some kind of bandaid fix I removed for being shit.

**Comment by Niknakflak** [comment]
The ships load completely fine, the problem only exists when moving the shuttle with a mapped docking port.  If you map load a ship that has no docking port pre-mapped, and spawn one in live on server, it works fine.  The problem is, if you used an area while mapping, a docking port spawned in won't move the area because of some conflict with docking ports and areas, but if you map a docking port in an area it should work and does so in regular mapping without the loader, but with the loader we have this bug.

I can assure you the docking port parameters were set perfectly and they didn't change when uploaded using the map placer.  Even if they were wrong, it wouldn't cut a ship up like that, it would cut off tiles across the whole ship, not like 2 on one side, and 3 on the other.  If that makes any sense.

**Comment by Niknakflak** [comment]
An update on this.  If you load huge maps with the map loader, it seems to load in "two parts".  Can't explain it really but just try and you'll see it.   It seems to cut off that second loaded part
